code/datums/status_effects/debuffs.dm

```diff
@@ -149,6 +149,29 @@
 /datum/status_effect/pacifism/on_remove()
 	REMOVE_TRAIT(owner, TRAIT_PACIFISM, id)
 
+// used to track if hitting someone with a cult dagger/sword should stamina crit.
+/datum/status_effect/cult_stun_mark
+	id = "cult_stun"
+	duration = 10 SECONDS // the when they stand back up, it will be over
+	var/mutable_appearance/overlay
+
+/datum/status_effect/cult_stun_mark/on_apply()
+	. = ..()
+	if(!. || !ishuman(owner))
+		return
+	overlay = mutable_appearance('icons/effects/cult_effects.dmi', "shield-cult", ABOVE_MOB_LAYER)
+	var/mob/living/carbon/human/H = owner
+	H.add_overlay(overlay)
+
+/datum/status_effect/cult_stun_mark/on_remove()
+	owner.cut_overlay(overlay)
+
+/datum/status_effect/cult_stun_mark/proc/trigger()
+	owner.adjustStaminaLoss(60)
+	owner.Silence(6 SECONDS) // refresh the silence
+	to_chat(world, "cowabunga")
```

Comment:
I too have everyone in the world hear cowabunga when I stab someone with a sword